addons/core/functions/fnc_getVehicleRadios.sqf

```diff
@@ -0,0 +1,37 @@
+#include "script_component.hpp"
+
+/*
+    Name: TFAR_fnc_getVehicleRadios
+
+    Author(s):
+        Jaffa
+
+    Description:
+        Gets a list of radios in the specified vehicle
+
+    Parameters:
+        0: Object - vehicle
+
+    Returns:
+         Array of LR radio arrays in format: 
+             0 - Object - Vehicle, 1 - String - Radio Settings ID
+         or empty array if no vehicle radios found
+
+    Example:
+        _radios = _vehicle call TFAR_fnc_getVehicleRadios;
+*/
+params ["_vehicle"];
+
+if (isNull _vehicle || {!(_vehicle call TFAR_fnc_hasVehicleRadio)}) exitWith {[]};//Vehicle is null or vehicle doesn't have LR Radio
+
+private _result = [[_vehicle, "gunner_radio_settings"],[_vehicle, "driver_radio_settings"],[_vehicle, "commander_radio_settings"],[_vehicle, "copilot_radio_setting"]];
+        
+private _turrets = [(typeof _vehicle), "TFAR_AdditionalLR_Turret", []] call TFAR_fnc_getVehicleConfigProperty;
+private _turrets = _turrets apply {[_vehicle, format ["turretUnit_%1_radio_setting",_x]]};
```

Comment:
Don't private again.

Reply:
Ohh, and this is wrong:
`TFAR_AdditionalLR_Turret` returns the following (example):
https://git.koffeinflummi.de/bwmod-team/bwa3-public/blob/master/bwa3_leopard2a6m/CfgVehicles.hpp#L71
This would result in:
`"turretUnit_[0,1]_radio_setting"` instead of e.g.: "turretUnit_0_radio_setting"
ref: https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/1.0/addons/core/functions/fnc_vehicleLr.sqf#L45